proto/storage/notifier.proto

```diff
@@ -27,6 +28,7 @@ message Notifier {
         AWSSecurityHub aws_security_hub  = 17;
         Syslog syslog                    = 18;
     }
+    Traits traits = 50;
```

Comment:
Why `50` and not `19`?

Also, could you add a comment for the notifier message with the next tag?

Reply:
That's not exactly correct - adding tag `1` to `oneof` block produces an error:

```
storage/notifier.proto:22:23: Field number 1 has already been used in "storage.Notifier" by field "id".
```

For the proto you are referencing, there are no other fields than the ones in `oneof` block - the other fields are in sub-messages, in which apparently tags are unique to the sub-message.

Reply:
🤯